modules/modules.mk

```diff
@@ -245,11 +245,20 @@ ifeq ($(FSI),yes)
   APPLICATION_DIR    := $(MOOSE_DIR)/modules/fsi
   APPLICATION_NAME   := fsi
 
-  DEPEND_MODULES     := tensor_mechanics navier_stokes
+  DEPEND_MODULES     := fluid_properties navier_stokes ray_tracing rdg tensor_mechanics
```

Comment:
need heat conduction for navier stokes?

Reply:
I am kind of puzzed. FSI requires `navier_stokes` and `tensor_mechanics`, 

and navier_stokes requires `fluid_properties` or whatever. 

Why we need to specify the requirements of `navier_stokes` in FSI? I would imagine it should be done automatically

Reply:
@fdkong - it is not done automatically, unfortunately. Hence why fsi will not compile standalone right now.

Reply:
Well, I think it can be done somewhat automatically if the ordering at the bottom is done correctly. Let me see.

Reply:
Done